apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -44,9 +44,13 @@ var ColorPickerPropertyRow = React.createClass({
       verticalAlign: 'top'
     };
     let colorPicker = this.state.displayColorPicker ? (
-      <ColorPicker type="sketch"
-                   color={this.state.value}
-                   onChangeComplete={this.handleColorChange}/>
+      <div style={{position: 'absolute'}}>
+        <div style={{position: 'fixed', top: '0', right: '0', bottom: '0', left: '0'}}
+             onClick={this.toggleColorPicker}></div>
```

Comment:
Hmm... on further investigation it looks like there are [a number](https://github.com/casesandberg/react-color/issues/67) [of issues](https://github.com/casesandberg/react-color/issues/93) [with this component](https://github.com/casesandberg/react-color/issues/20) and auto-closing... and [under the hood it's doing what you do here](https://github.com/casesandberg/react-color/blob/aa3e4bab0dcb0eaaa9cdc1595cc295711867b921/examples/Button.js#L24).  So this is fine.  Ignore me :grin:.